framework/src/org/apache/cordova/CordovaWebViewClient.java

```diff
@@ -171,8 +171,13 @@ public void onPageStarted(WebView view, String url, Bitmap favicon) {
     public void onPageFinished(WebView view, String url) {
         super.onPageFinished(view, url);
         // Ignore excessive calls.
-        if (!isCurrentlyLoading) {
-            return;
+        //     On KitKat, loading about:blank does not invoke WebViewClient#onPageStarted
+        //     So do not check isCurrentLoading if app is running on KitKat
+        //         https://code.google.com/p/android/issues/detail?id=65701
+        if (android.os.Build.VERSION.SDK_INT != android.os.Build.VERSION_CODES.KITKAT){
```

Comment:
I just tested it on Lollipop and confirmed onPageStarted was NOT called for about:blank loading so my patch does not work on Lollipop. The same bug is still there in Android > KITKAT.